app/controllers/home_controller.rb

```diff
@@ -1,6 +1,12 @@
 class HomeController < ApplicationController
   def index
-    redirect_to admin_organisations_path if current_user.super_admin?
+    if current_user.super_admin?
```

Comment:
This doesn't look idomatic to me, but I'm not sure what the right "Rails way" to do it is.

(From talking IRL, there were some linting problems around doing `redirect and return`, but worth having a quick read around early returns from a controller method).

Reply:
Initially, this was written using `redirect and return` and all the current tests passed. 
However, the linter prefers `redirect && return` and this caused some tests (and some unrelated tests to fail).

I have time-boxed, researched and had a play around with this but changing from `redirect and return` to `redirect && return` still caused tests to fail. 

- Currently (at time of writing this) I have refactored the conditional to look like:
`  return redirect_to admin_organisations_path if current_user.super_admin?
    redirect_to setup_index_path unless current_organisation.ips.present?` which passes the tests. 
**Still I'm open to insight/thoughts into what may be causing `redirect && return` to fail some tests.**

Reply:
Yeah, I'm not sure how to resolve this - think you're right to timebox and move on.